app/models/manageiq/providers/kubevirt/inventory/parser.rb

```diff
@@ -111,18 +111,56 @@ def process_node(object)
     )
   end
 
-  def process_stored_vms(objects)
+  def process_offline_vms(objects)
     objects.each do |object|
-      process_stored_vm(object)
+      process_offline_vm(object)
     end
   end
 
-  def process_stored_vm(object)
+  def process_offline_vm(object)
+    # Get the basic information:
+    uid = object.metadata.uid
+    name = object.metadata.name
+    domain = object.spec.template.spec.domain
+
+    # Process the domain:
+    vm_object = process_domain(domain, uid, name)
+
+    # The power status is initially off, it will be set to on later if the live virtual machine exists:
+    vm_object.raw_power_state = 'off'
+  end
+
+  def process_live_vms(objects)
+    objects.each do |object|
+      process_live_vm(object)
+    end
+  end
+
+  def process_live_vm(object)
     # Get the basic information:
     uid = object.metadata.uid
     name = object.metadata.name
     domain = object.spec.domain
 
+    # Get the identifier of the offline virtual machine from the owner reference:
+    owner = find_owner(object, 'OfflineVirtualMachine')
+    unless owner
+      _log.info(
```

Comment:
this is not recognized here - you should add:

`  include Vmdb::Logging`

Reply:
Thanks @masayag, did that in the latest version of the pull request.